userspace/libsinsp/test/filterchecks/evt.cpp

```diff
@@ -66,4 +66,18 @@ TEST_F(sinsp_with_test_input, EVT_FILTER_rawarg_str)
 	sinsp_evt* evt = add_event_advance_ts(increasing_ts(), 1, PPME_SYSCALL_OPEN_E, 3, path.c_str(),
 					      (uint32_t)0, (uint32_t)0);
 	ASSERT_EQ(get_field_as_string(evt, "evt.rawarg.name"), path);
+}
+
+TEST_F(sinsp_with_test_input, EVT_FILTER_cmd_str)
+{
+	add_default_init_thread();
+	
+	open_inspector();
+
+	std::string cmd = "BPF_PROG_LOAD";
+	int fd = 1;
+
+	sinsp_evt* evt = add_event_advance_ts(increasing_ts(), 1, PPME_SYSCALL_BPF_2_X, 2, fd, "BPF_PROG_LOAD");
```

Comment:
```suggestion
	sinsp_evt* evt = add_event_advance_ts(increasing_ts(), 1, PPME_SYSCALL_BPF_2_X, 2, fd, (uint32_t)PPM_BPF_PROG_LOAD);
```

Pulled the changes and ran with the `sinsp-example` test binary? Something seems to be off, would you mind taking another look? Happy to continue the review then.